app/javascript/packs/pdf_viewer.js

```diff
@@ -3,13 +3,11 @@
 import { EmbedThis } from '../src/modules/embed_this.js';
 import { PopupPanels } from '../src/modules/popup_panels.js';
 import CommonViewerBehavior from '../src/modules/common_viewer_behavior.js';
-import PDFViewer from '../src/modules/pdf_viewer.js';
 import Fullscreen from '../src/modules/fullscreen.js';
 
 $(document).ready(function() {
   CommonViewerBehavior.initializeViewer();
   PopupPanels.init();
   EmbedThis.init();
-  PDFViewer.init();
```

Comment:
remove pdf.js import and init

Reply:
I'm not sure you can have the legacy pdf viewer work if you modify this file.  I think you either need to remove the legacy pdf component or make a new pack for the new pdf viewer.

Reply:
Indeed it would not.  That is why I moved the original version of this file to `app/javascript/packs/legacy_pdf_viewer.js` (see above) and reference that pack in the `LegacyPdfComponent`.  If/when we remove the Legacy component, we can just delete the entire legacy pack file (as described in the comments in `app/components/embed/legacy_pdf_component.rb`).